src/env.spec.ts

```diff
@@ -6,7 +6,61 @@
  * Copyright 2019 Datadog, Inc.
  */
 
-import { getConfig, defaultConfiguration, setEnvConfiguration, forceExcludeDepsFromWebpack } from "./env";
+import {
+  getConfig,
+  defaultConfiguration,
+  setEnvConfiguration,
+  forceExcludeDepsFromWebpack,
+  hasWebpackPlugin,
+} from "./env";
+
+describe("hasWebpackPlugin", () => {
+  it("returns false when the serverless.yml plugin(s) object is not defined", () => {
```

Comment:
Nit: I think the object is always called `plugins` so I would suggest `plugin(s)` => `plugins`

Reply:
(same in the other tests)